api/collections.js

```diff
@@ -211,7 +211,7 @@ function removeQuoteFromMyCollection(req, res, next) {
       }
       return quoteCollection.removeQuote(req.body.quoteId, req.user.id)
         .then(status => res.status(HTTPStatus.OK).send(status))
-        .error((e) => res.status(HTTPStatus.BAD_REQUEST).send(e.message));
+        .error(e => res.status(HTTPStatus.BAD_REQUEST).send(e.message));
```

Comment:
Let's bring in some _standard_ conventions here:

- `e` means event
- `err` means error

Also you might sometimes find `reason` as `catch` parameter name because Promise could technically be rejected by non-error value bad that's code smell...